src/views/Message/Event.tsx

```diff
@@ -38,7 +29,7 @@ const EventPage = () => {
       setEvent(result.data.event_name);
     };
     fn();
-  }, [getAccessTokenSilently]);
+  }, [getAccessTokenSilently, token]);
```

Comment:
Nit: For this we could have it only depend on `token` and not `getAccessTokenSilently`. And then update the header config to be

```
const config = {
        headers: {
          Authorization: `Bearer ${token}`,
        },
      };
```

Then can also remove the `getAccessTokenSilently` import as well.